pfrl/experiments/train_agent_async.py

```diff
@@ -207,11 +208,11 @@ def train_agent_async(
             other thread raised an excpetion. The train will be terminated and
             the current agent will be saved.
             If set to None, a new Event object is created and used internally.
+        use_shared_memory (bool): Prevent using shared memory.
```

Comment:
```suggestion
        use_shared_memory (bool): Share memory amongst asynchronous agents.
```
If true, then share memory, right? So we shouldn't say "Prevent"?